pkg/types/v1/luet_test.go

```diff
@@ -42,7 +46,17 @@ var _ = Describe("Types", Label("luet", "types"), func() {
 	Describe("Luet", func() {
 		It("Fails to unpack without root privileges", Label("unpack"), func() {
 			image := "quay.io/costoolkit/releases-green:cloud-config-system-0.11-1"
-			Expect(luet.Unpack(target, image)).NotTo(BeNil())
+			Expect(luet.Unpack(target, image, false)).NotTo(BeNil())
+			image := "docker.io/library/alpine"
```

Comment:
This should be its own test IMO.

```suggestion
		})
		It("Unpacks local image", Label("unpack", "root"), func() {
			image := "docker.io/library/alpine"
```